dss/config.py

```diff
@@ -97,20 +99,57 @@ def get_gs_bucket() -> str:
         return Config._GS_BUCKET
 
     @staticmethod
-    def _clear_cached_config():
+    def get_allowed_emails() -> str:
+        if Config._ALLOWED_EMAILS is None:
```

Comment:
We're using Service Account credentials when testing on Travis but, for the moment, only allowing email addresses ending in `chanzuckerberg.com ucsc.edu broadinstitute.org ebi.ac.uk` when live.